frontend/src/components/ui/PredictTable.tsx

```
@@ -0,0 +1,135 @@
+                <button type="button" onClick={handleSubmit} className={submitButtonStyle}>Submit</button>   */}
+            </div>
+            {table[0].length>0 && 
+                <div>
+                    <table className="w-full m-2 mt-4 block border-collapse table-fixed">
+                        <tr className="bg-slate-800 text-slate-200">
+                            <th className="px-4 py-3 w-[150px] text-left font-medium">Campus</th>
+                            <th className="px-4 py-3 w-[400px] text-left font-medium">Branch</th>
+                            <th className="px-4 py-3 text-right font-medium">Marks</th>
+                        </tr>
+                        {(Array.from({length: tableRange[1]-tableRange[0]+1 }, (_, i)=>i+tableRange[0])).map((e)=>(
+                            <tr className="border-b border-slate-700">
```

Biome (2.3.13)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)